src/components/ChatEntry.js

```diff
@@ -1,22 +1,52 @@
-import React from 'react';
+import React, {useState} from 'react';
 import './ChatEntry.css';
 import PropTypes from 'prop-types';
+import TimeStamp from './TimeStamp';
 
 const ChatEntry = (props) => {
+  const [heartButton, setheartButton] = useState('🤍');
```

Comment:
Awesome! Small nit: the `h` in the setter state value should be capitalized to `setHeartButton`